### DISCUSSION TITLE
compression between manager and search node

### DISCUSSION THREAD

**Discussion by palichx** [Security-Onion-Solutions/securityonion#4063] [Unsupported Versions]
In my setup i see diference betwiin incoming logstash trafic rate from sensor to manager and between manager search nodes.
Sensors to manager traffic rate is 30mbps
Manager to serach nodes traffic rate is 300mbps.
![image](https://user-images.githubusercontent.com/80513304/116685355-de171d00-a9ba-11eb-851a-062092f1a09d.png)

As i undestund, compression level between sensor and manager is 3. (default compresson in filebeat config).

looks like there is no compression between manager node and search nodes.  Is there a way to configure compresson level on this communication? 
thanks!

**Comment by TOoSmOotH** [comment]
The manager does not talk directly to the search nodes. The outbound traffic from that container is to REDIS which is local. The search nodes then use the REDIS queue to load balance pulling and ingesting events. The traffic between logstash and REDIS is not compressed so storage nodes will pull from the REDIS queue uncompressed.
